### DISCUSSION TITLE
API Documentation?

### DISCUSSION THREAD

**Discussion by Nanogramai** [UltimaHoarder/UltimaScraper#1986] [General]
Hiii, 

I am an OnlyFans creator so I am not looking to scrape/download content. I am wanting to build scripts to automate some of my tasks. IE - Auto scheduling messages and posts, auto messaging subscribers who are about to expire, creating lists based on activity or spending, etc.

Could you please create a Documentation for your API?

**Comment by DIGITALCRIMINAL** [comment]
This wrapper is nowhere near finished yet, and I break changes every other commit.

I'll hold off until some GPT software can auto-generate it for me.

**Comment by Nanogramai** [comment]
Thanks for getting back to me! With that said, would you be interested in
hof feeling your services to us? I’m very new to software development, but
if you would be able to provide me with the endpoints I need, I would be
happy to hire you for your help!

Essentially, I run an agency for OnlyFans creators. Currently, we are
managing five creators all within the top 0% of OnlyFans, but we are doing
everything manually except for their social media which I have been able to
build Python scripts for. My goal is to be able to automate some of the
futile tasks involved with running an OnlyFans account.

Please let me know if you are interested and we can schedule a call
sometime in the next couple weeks!

**Comment by Nanogramai** [comment]
Offering your services **
